src/org/javarosa/core/model/utils/DateUtils.java

```diff
@@ -270,8 +272,7 @@ public static String format (DateFields f, String format) {
 				} else if (c == 'n') {	//numeric month
 					sb.append(f.month);
 				} else if (c == 'b') {	//short text month
-					String[] months = new String[] {"Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"};
-					sb.append(months[f.month - 1]);
+                    sb.append(DateTimeFormat.forPattern("MMM").print(new DateTime(DateUtils.getDate(f))));
```

Comment:
Yeah, I'd be inclined to just be consistent with the rest of the file and use tabs.